src/components/profile/Profile.js

```diff
@@ -0,0 +1,203 @@
+              <input
+                role="date"
+                className="w-full px-3 py-2 leading-tight text-gray-700 border rounded shadow appearance-none focus:outline-none focus:shadow-outline"
+                type="date"
+                name="date"
+                onChange={handleInputChange}
+              />
+            </div>
+            <div className="mb-4">
+              <label className="block mb-1 text-sm font-bold text-gray-700">Currency</label>
+              <select
+                role="currency"
+                className="w-full px-3 py-2 leading-tight text-gray-700 border rounded shadow appearance-none focus:outline-none focus:shadow-outline"
+                onChange={handleInputChange}
+                name="currency"
+              >
+                <option>select currency</option>
+                <option value="Local">Local Currency</option>
+                <option value="Dollars">US Dollars</option>
+                <option value="Euros">Euro Currency</option>
+              </select>
+              <input />
+            </div>
+            <div className="mb-4">
+              <label className="block mb-2 text-sm font-bold text-gray-700">Department</label>
+              <input
+                role="departement"
+                className="w-full px-3 py-2 leading-tight text-gray-700 border rounded shadow appearance-none focus:outline-none focus:shadow-outline"
+                type="text"
+                onChange={handleInputChange}
+                name="departement"
+              />
+            </div>
+            <div className="mb-4">
+              <label className="block mb-2 text-sm font-bold text-gray-700">
+                Are you a manager ?
+              </label>
+              <div className="flex justify-center">
+                <div className="form-check form-check-inline">
+                  <input
+                    className="float-left w-4 h-4 mt-1 mr-2 align-top transition duration-200 bg-white bg-center bg-no-repeat bg-contain border border-gray-300 rounded-sm appearance-none cursor-pointer form-check-input checked:bg-blue-600 checked:border-blue-600 focus:outline-none"
+                    type="checkbox"
+                    id="inlineCheckbox1"
+                    value="true"
+                    onChange={handleCheck}
+                  />
+                  <label className="inline-block text-gray-800 form-check-label">True</label>
+                </div>
+              </div>
+            </div>
+            <div className="flex items-center justify-between">
+              <button
+                role="UpdateButton"
+                className="px-4 py-2 font-bold text-white rounded hover:bg-blue-700 focus:outline-none focus:shadow-outline"
+                type="submit"
+                id="updateButton"
+                style={{ backgroundColor: '#D2691E' }}
+              >
+                Update
```

Comment:
1. You should prepopulate in the input field with the current information as it is updating
2. You should consider responsiveness because when it is on mobile it gets messy